src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -22,22 +22,22 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends"), getAppointmentSet("23:59 SUN", "00:00 MON")),
+                getTagSet("friends"), getAppointmentSet("12:00-13:00 SUN", "00:00-01:00 MON")),
```

Comment:
Not too sure how to cover these in tests, would just spam checking each person being valid suffice?